corehq/apps/linked_domain/tests/test_linked_apps.py

```diff
@@ -170,15 +160,7 @@ def test_override_translations(self):
 
         self.linked_app.master = self.master1.get_id
 
-        copy = self.master1.make_build()
-        copy.save()
-        self.addCleanup(copy.delete)
-
-        self.master1.save()  # increment version number
-        copy1 = self.master1.make_build()
-        copy1.is_released = True
-        copy1.save()
-        self.addCleanup(copy1.delete)
```

Comment:
Is this missing a second master build?

Reply:
The test does pass either way, but I'll add another build back just in case.

Reply:
0d7990f35fba6405cbc860d5829de3a4ea4db929